src/components/EventCreateDialog.vue

```diff
@@ -0,0 +1,135 @@
+<!--
+  - @copyright Copyright (c) 2021 Jakob Röhrl <jakob.roehrl@web.de>
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  - @author Jakob Röhrl <jakob.roehrl@web.de>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Modal class="card-selector" @close="close">
+		<div class="modal-scroller">
+			<div v-if="!creating && !created" id="modal-inner" :class="{ 'icon-loading': loading }">
+				<h3>{{ t('calender', 'Create a new event') }}</h3>
+			</div>
+		</div>
+	</Modal>
+</template>
+
+<script>
+
+import Modal from '@nextcloud/vue/dist/Components/Modal'
+// import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
+// import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'
+
+export default {
+	name: 'EventCreateDialog',
+	components: {
+		// EmptyContent,
+		Modal,
+		// Multiselect,
+	},
+	props: {
+		title: {
+			type: String,
+			default: '',
+		},
+		description: {
+			type: String,
+			default: '',
+		},
+		action: {
+			type: String,
+			default: t('calender', 'Create event'),
```

Comment:
```suggestion
			default: t('calendar', 'Create event'),
```

Reply:
@tcitworld Thanks for looking over it. 
Do you have an idea, why my script talk.js doesn't load any more? I get a 404.
Is this wrong: https://github.com/nextcloud/calendar/blob/48184df104966ed74e4ac71b3b1a31ecd7c81f06/webpack.js#L19